final_prep/mod_zero_hero.rb

```diff
@@ -57,28 +105,66 @@
 #   - address (hash with following key/values: number , street , state, zip)
 
 
+scary_monster = {'name' => 'gus', 'smell' => 'bad', 'weight' => 150, 'citiesDestroyed' => ['new york', 'london'], 'luckyNumbers' => [4, 5, 6], 'address' => {'number' => 50, 'street' => 'doty', 'state' => 'wi', 'zip' => 50220}}
```

Comment:
It's easier to read and digest hashes when the key/value pairs are formatted on multiple lines.